doc/examples/simulate_dki.py

```diff
@@ -0,0 +1,83 @@
+"""
+==========================
+DKI MultiTensor Simulation
+==========================
+
+In this example we show how someone can simulate the diffusion tensor,
```

Comment:
I think it's more common to say we show one can instead of someone. Is there actually a native english speaker we can poke for that?